helpdesk_type_reminder/models/helpdesk_ticket_type.py

```diff
@@ -0,0 +1,9 @@
+# © 2020 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import fields, models
+
+
+class HelpdeskType(models.Model):
+    _inherit = "helpdesk.ticket.type"
+
+    include_reminder = fields.Boolean(string='Include in reminder')
```

Comment:
@lmiguens-solvos why should default value be?

Reply:
I think it should have the same default value as the field reminder_active of helpdesk.ticket (reminder_active = fields.Boolean (string = 'Reminder include', default = True)) right?